src/cpp/CMakeLists.txt

```diff
@@ -56,6 +56,8 @@ ov_genai_build_jinja2cpp()
 
 # Library
 
+find_package(OpenVINO REQUIRED COMPONENTS Runtime Threading)
```

Comment:
maybe you can change main [find_package](https://github.com/openvinotoolkit/openvino.genai/blob/master/CMakeLists.txt#L52-L60) call in root cmake like it's done in tokenizers?
https://github.com/openvinotoolkit/openvino_tokenizers/blob/5ccd56de5a7da65ad7ea7c5c42cec55be97df312/CMakeLists.txt#L70-L76

Reply:
Good idea, done